pages/common/wget.md

```diff
@@ -7,6 +7,10 @@
 
 `wget {{https://example.com/foo}}`
 
+- Mirror all listed files within a directory and its sub-directories only (no parent):
```

Comment:
Please move this below the "Download a full website" example.

And also, the word "mirror" can be a bit confusing to newbies. Let's use the same wording as the "Download a full website" example. And update the description to use the directory name "foo". Something like - `Download only foo and its sub-directories without its parent directories.`